src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -143,10 +145,10 @@ internal static class MvcCoreLoggerExtensions
 
         static MvcCoreLoggerExtensions()
         {
-            _actionExecuting = LoggerMessage.Define<string>(
-                LogLevel.Debug,
+            _actionExecuting = LoggerMessage.Define<StringBuilder, string>(
```

Comment:
Can you make this a string? It's just really odd to see StringBuilder show up here. We already know that it's stringlike, and we already know we're going to log it.